x/application/types/message_delegate_to_gateway.go

```diff
@@ -37,6 +37,12 @@ func (msg *MsgDelegateToGateway) GetSignBytes() []byte {
 	return sdk.MustSortJSON(bz)
 }
 
+func (msg *MsgDelegateToGateway) NewDelegateeChangeEvent() *EventDelegateeChange {
+	return &EventDelegateeChange{
+		AppAddress: msg.AppAddress,
```

Comment:
Am against this as it is irrelevent to its usage

Reply:
Perhaps this is true of the current usage. However, imagine building an indexer for this, for example. It would be extremely convenient for the event to include the extra detail of which gateway the given application is (re)delegating to:

1. It disambiguates the different types of the redelegation event (i.e. delegation, redelegation, undelegation).
2. It removes the need to make a query for the related effected on-chain state.